doc/adr/0002-use-typescript.md

```diff
@@ -0,0 +1,23 @@
+# 2. Use TypeScript
+
+Date: 2020-09-04
+
+## Status
+
+Accepted
+
+## Context
+
+We need a way to describe the structure of Javascript objects for readers of the code to be able to understand the structure.
+And we need a way to detect null references at build time Javascript.
+Our Javascript passes though a build step.
+Our Javascript is maintained by people who did not write the code.
```

Comment:
Hoeveel code wordt onderhouden door mensen die de code niet hebben geschreven?

Reply:
my guesstimate would be > 90%